src/libpmemobj/stats.c

```diff
@@ -95,16 +95,21 @@ struct stats *
 stats_new(PMEMobjpool *pop)
 {
 	struct stats *s = Malloc(sizeof(*s));
+	if (s == NULL) {
+		ERR("!Malloc");
+		goto error_alloc;
```

Comment:
There's no point in going to error_alloc - just return NULL here and then you won't need the change below.